components/dashboard/src/workspaces/workspace-model.ts

```diff
@@ -145,8 +145,12 @@ export class WorkspaceModel implements Disposable, Partial<GitpodClient> {
     }
 
     protected isActive(info: WorkspaceInfo): boolean {
+        const lastSessionStart = WorkspaceInfo.lastActiveISODate(info);
+        const twentyfourHoursAgo = new Date(Date.now() - 1000 * 60 * 60 * 24).toISOString();
```

Comment:
In `timeutil.ts` there are helpers for this - `hoursBefore(lastSessionStart, 24)`